crates/libcgroups/Cargo.toml

```diff
@@ -25,7 +25,7 @@ procfs = "0.15.1"
 oci-spec = { version = "~0.6.3", features = ["runtime"] }
 fixedbitset = "0.4.2"
 serde = { version = "1.0", features = ["derive"] }
-rbpf = { version = "0.2.0", optional = true }
+rbpf = { version = "0.3.0", optional = true }
```

Comment:
Is this accidental ? `rbpf` only has `0.2.0`

Reply:
🤦‍♂